ext/fg/js/popup.js

```diff
@@ -243,6 +248,25 @@ class Popup {
         return popupPreparedPromise;
     }
 
+    _onFrameLoad() {
+        if (!(this._injectPromise !== null && this._injectPromiseComplete)) { return; }
+        this._resetFrame();
+    }
+
+    _resetFrame() {
+        const parent = this._container.parentNode;
+        if (parent !== null) {
+            parent.removeChild(this._container);
+        }
+        this._container.removeAttribute('src');
+        this._container.removeAttribute('srcdoc');
+
+        this._containerSecret = null;
+        this._containerToken = null;
```

Comment:
These don't seem to be used anywhere else, did you mean `_messageToken` perhaps?

Reply:
Oops; leftovers from a different commit I was based on. You'll probably see these in a future commit, but for now, they aren't used.
25243f33207ba9d853afdbbbc9633ff02a117a79